QHyper/problems/TSP.py

```diff
@@ -0,0 +1,135 @@
+import itertools
+
+import numpy as np
+
+from .problem import Problem
+
+
+class TSP:
```

Comment:
I see now that this class needs some refactoring, e.g., not all of the parameters in init are needed, but I'll take care of that later due to time restrictions now :(.